FreeRTOS/FreeRTOS
# Update FreeRTOS+TCP to latest version V4.3.1

Description
-----------
Update FreeRTOS+TCP to latest version V4.3.1 
The changes are tested in the CI workflow.

Test Steps
-----------


Checklist:
----------


- ~[] I have modified and/or added unit-tests to cover the code changes in this Pull Request.~

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
